src/v2/Components/ArtworkFilter/ArtworkFilters/SizeFilter.tsx

```diff
@@ -146,6 +146,7 @@ export const SizeFilter: React.FC<SizeFilterProps> = ({ expanded }) => {
       sizes,
       height: "*-*",
       width: "*-*",
+      reset: false,
```

Comment:
Good find! Do you have a link to where `reset: false` is thread through and used? Wondering how this ends up fixing up the problem even though your screen recording clearly demonstrate that it does.

Reply:
The `reset` field used in useEffect hook used for reset custom size values in component.
When a user reset filters for the first time, `reset` field set true. Then, when the user click reset for the second time, `reset` field again set true, but in component it seems like `reset` don't updated, so effect doesn't work and custom size doesn't reset anymore.

The alternative way to fix this bug is update all filters and set `reset: false` when custom size reset. I mean 
```
useEffect(() => {
    // if filter state is being reset, then also clear local input state
    if (reset) {
      setCustomSize({ height: ["*", "*"], width: ["*", "*"] })
      setFilters?.({
          ...currentlySelectedFilters?.(),
          reset: false
      }, { force: false })
    }
  }, [reset])
```

Reply:
But I think the current solution is cleaner 🙂